todo_app/templates/index.html

```diff
@@ -14,21 +14,17 @@ <h2>Items</h2>
         <table>
           <thead>
             <tr>
-          {% for header in headers %}
-          <th>{{header}}</th>
-          {% endfor %}
-        </tr>
-        </thead>
 
-        <tbody>
-          {% for result in results %}
-          <tr>
-            {% for k, val in result.items() %}
-              <td>{{val}}</td>
+            </tr>
+          </thead>
+          <tbody>
+            {% for todo in todos %}
+            <li>{{todo.id}} - "To Do" - {{todo.name}}</li>
+            <form method="post" action="/">
+              <button type="submit" name="cardid" value={{todo.id}}> Move to Done </button>
+            </form>
```

Comment:
It would be good to display done items as well, with something that makes it clear which items are done and which are still to do.

Reply:
I've added the done items